### ISSUE TITLE
[Light Gallery] Error when opening image in new tab

### ISSUE THREAD

**Issue by rowi68** [JoomGalleryfriends/JG4-dev#315]
When try to open an image with right click in a new tab, thousands of "hieroglyphs" appear as follows:

![2024-12-28_08h05_32](https://github.com/user-attachments/assets/fe038a2b-7c70-4893-90bc-c05c26a9652d)

**Comment by Elfangor93** [comment]
Please provide more input in order to help us localize the issue...

Where do you perform the right click, which view?
How does the image url looks like at the image you want to do the right click?

> open an image with right click in a new tab
Which context menu are you exactly taking here?

**Comment by rowi68** [comment]
I did right-click with mouse on an image

**url:** .../index.php?option=com_joomgallery&view=image&id=2744&Itemid=647
![2024-12-29_14h56_07](https://github.com/user-attachments/assets/7d70552b-3cb0-47dd-a40f-f90e6ecd8c2e)


View in new tab appears like this:

![399144069-fe038a2b-7c70-4893-90bc-c05c26a9652d](https://github.com/user-attachments/assets/1db3659f-7f52-43b4-89ff-4819dc57b334)

**url:** .../index.php?option=com_joomgallery&view=image&id=2744&format=raw&type=detail&Itemid=647

J5.2.2 | PHP 8.3

**Comment by Elfangor93** [comment]
> I did right-click with mouse on an image

Where exactly? In detail view? In lightgallery? Frontend or Backend?

**Comment by rowi68** [comment]
Yes , in frontend detail view. But i just also tested in backend and lightbox, everywhere same result.
I use Browser Firefox, but have tested also with Google Chrome Browser.

Browser console displays:
![2024-12-29_16h08_39](https://github.com/user-attachments/assets/d03792e3-e641-4ee2-b850-e323d1aa02b7)

**Comment by Elfangor93** [comment]
Does this happen specifically to this very image or is it in general?
I am asking so much because I can not reproduce this issue in my installations...

**Comment by rowi68** [comment]
It is in general. All images, also in backend.
Follow my system information...
![2024-12-29_18h42_56](https://github.com/user-attachments/assets/38a54f50-5934-420d-b215-52f1a11c6fe9)

**Comment by hansdampf-123** [comment]
On my site that happens if you set:
Category View - Category Images - Image link to Link to Light Gallery 
If I then right click and open in new tap. The funny screen appears.

**Comment by reilldesign** [comment]
That is currently the intention. The images are not output with their URL paths but with the PHP syntax to prevent image links from being shared. These are not resolved in a new tab. But an optional download button would be possible.

**Comment by Elfangor93** [comment]
Close this as it is function not bug.
